web/profiles/joinup/joinup.info.yml

```diff
@@ -17,6 +17,7 @@ dependencies:
   - file
   - help
   - image
+  - joinup_document
```

Comment:
I'd prefer collection_document as a name for the module, as it better represents the relation to the group.